src/docker/base/alpine/Dockerfile

```diff
@@ -49,7 +49,11 @@ RUN pip install -U --no-cache-dir \
     setuptools==${SETUPTOOLS_VERSION} \
     ${FLWR_PACKAGE}==${FLWR_VERSION}
 
-FROM python:${PYTHON_VERSION}-${DISTRO}${DISTRO_VERSION} as base
+FROM python:${PYTHON_VERSION}-${DISTRO}${DISTRO_VERSION} AS base
+
+# Upgrade system Python pip and setuptools
+# hadolint ignore=DL3013
+RUN pip install -U --no-cache-dir pip setuptools
```

Comment:
Shouldn't we fix it to a particular version to make the build reproducible?

Reply:
Here we upgrade the pip and setuptools version of the global installed Python version. When we install flwr, we install it in a virtual env and there we install the specific version:
https://github.com/adap/flower/blob/a20e4c9a13681587aba2760bfdad23e5343af1f7/src/docker/base/ubuntu/Dockerfile#L62-L65